### DISCUSSION TITLE
ssr for development only

### DISCUSSION THREAD

**Discussion by BhaskaranR** [frandiox/vite-ssr#126] [Q&A] [answered]
Hi, Newbie here. Thanks for the good work. how do i build the client only for production use? i would like to use ssr only for development. We are using spring boot to serve content in production. i don't want the manifest file and server side files to built for production. 

If I ```vite build``` i get this exception.
```
cp: dist/client: No such file or directory
 ELIFECYCLE  Command failed with exit code 1.
```

Appreciate any help.

**Comment by mvrlin** [comment, marked as answer]
@BhaskaranR you can just use `vite build`
